docs/ca-signed-workflow.md

```diff
@@ -99,7 +99,8 @@
    docker tag hello-world:latest $server/hello-world:v1
    docker push $server/hello-world:v1
    ```
-9. Sign the image with an external certificate bundle (`$certBundlePath`) including the intermediate certificates and a root certificate in PEM format. You may fetch the certificate bundle from your CA official site.
+9. Sign the image with an external certificate bundle (`$certBundlePath`) including the in PEM format. Check that the certificates in the certificate bundle are arranged in the correct order: starting from the first intermediate certificate that signed the leaf certificate and ending with the root certificate.
```

Comment:
could you add an example of the PEM encoded certificate chain file? Example text:

The certificates must be concatenated in order so that each directly certifies the one preceding. The following example contains two certificates: one is intermediate certificate, another is root certificate, but your certificate chain might contain more or fewer.

```
-----BEGIN CERTIFICATE-----
Base64–encoded intermediate certificate
-----END CERTIFICATE-----
-----BEGIN CERTIFICATE-----
Base64–encoded root certificate
-----END CERTIFICATE-----
```

Reply:
Updated